### PR TITLE
Updated bundles with comparator errors

### PR DESCRIPTION
**Pull request by trancexpress**
Changes to ecj are for handling of switch on String.

See: https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/issues/2595

## Author checklist

- [ ] I have thoroughly tested my changes
- [ ] The change is following the [coding conventions](https://wiki.eclipse.org/Platform/How_to_Contribute#Coding_Conventions)
- [ ] I have signed the [Eclipse Contributor Agreement (ECA)](https://www.eclipse.org/legal/ECA.php)

### CODE REVIEW

File: org.eclipse.jdt.debug.tests/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.jdt.debug.tests; singleton:=true
-Bundle-Version: 3.12.550.qualifier
+Bundle-Version: 3.12.650.qualifier
```
Comment (akurtakov) [inline org.eclipse.jdt.debug.tests/META-INF/MANIFEST.MF:5]: This looks wrong. I don't know how 550 ended up in master (IMO it's failure from merging BETA_JAVA* branch FYI @jarthana ) . The bundle version should be 3.12.600 here.

File: org.eclipse.jdt.debug.tests/pom.xml [hunk 2]
```diff
@@ -18,7 +18,7 @@
   </parent>
   <groupId>org.eclipse.jdt</groupId>
   <artifactId>org.eclipse.jdt.debug.tests</artifactId>
-  <version>3.12.550-SNAPSHOT</version>
+  <version>3.12.650-SNAPSHOT</version>
```
Comment (akurtakov) [inline org.eclipse.jdt.debug.tests/pom.xml:21]: Version should be 3.12.600.

**Review by akurtakov — CHANGES_REQUESTED** [review]
Please fix the o.e.jdt.debug.tests version.

**Review by akurtakov — APPROVED** [review]
Looks good to me now

**Comment by trancexpress** [comment]
> Please fix the o.e.jdt.debug.tests version.

Thanks for taking a look @akurtakov !

**Pull request merged**